deploy/manifests/agent/README.md

```diff
@@ -0,0 +1,17 @@
+# Agent
+
+The manifests used to deploy the Portainer agent inside a Kubernetes cluster.
+
+To deploy an Edge agent inside your Kubernetes cluster, it is recommended to follow the instructions available inside your Portainer instance.
+
+# Usage
+
+## Deploy the Portainer agent and access it via an external load balancer
+
+If your cloud provider supports external load balancers, you can use the following command to deploy the regular Portainer agent (not Edge):
+
+```
+kubectl ... apply -f portainer-agent.yaml
```

Comment:
Should probably use the new filename here.

Reply:
Fixed in latest commit :)